src/clarity-angular/buttons/_buttons.clarity.scss

```diff
@@ -215,6 +241,64 @@ $clr-button-appearance-map: (
         @include generateButton();
         @include clr-getStylesFromMap($clr-button-appearance-map, form);
         @include populateButtonProperties(default);
+    }
+
+    .btn-group {
+        //.btn-outline-<state> is the new BS4 naming convention
+        //TODO: we support both the previous one and the new one but we will deprecate the old ones from the example in 0.5.0
+        &.btn-secondary > .btn,
```

Comment:
I actually am still not sure about it. Button groups use the same class names as buttons. If we change button groups to use the new naming convention (`.btn-group--primary` instead of `.btn-group-primary`) won't that create more confusion? Users will have to remember 2 different classnames which do the same thing.